scripts/dropstat

```diff
@@ -30,7 +30,8 @@ try:
         sys.path.insert(0, modules_path)
         sys.path.insert(0, test_path)
         import mock_tables.dbconnector
-        socket.gethostname = lambda : 'sonic_drops_test'
+        socket.gethostname = lambda: 'sonic_drops_test'
+        os.getuid = lambda: 27
```

Comment:
what is the 27?

Reply:
It's a dummy value to ensure that the uid for the tests is deterministic. Should I make this a variable somewhere or is a comment OK?

Reply:
I'm confused. Is it always ```27``` ? What will happen if I ran ```dropstat``` in pytest or from shell directly?

Reply:
This code will only run during the unit tests. It is always 27 so that the unit tests can always check the same number when verifying the output and always delete the same number when it's cleaning up for future tests.

Reply:
Sorry for late reply. What the uid will be if I call ```show dropcounters counts``` in ansible?  ```show dropcounters counts``` will call ```dropstat -c show``` inside.

Reply:
`ansible` has root access so I believe it would just be `0`. Unless the unit test flag is set, which should only be when sonic-utilities unit tests are running, `os.getuid` will not be overwritten.

Reply:
What's the uid if I call ```show dropcounters counts``` from shell?

Reply:
Whatever admin's uid is.

Reply:
Thanks. I got it. The updated code will create an individual dir named ```/tmp/dropstat-{uid}``` for each user instead of using the same dir.  Make sense to me.